Src/Plugins/Tests.Security/ReflectionTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class ReflectionTests
+    {
+        [Fact]
+        public void All101Rules_ShouldHaveValidators()
```

Comment:
>All101Rules_ShouldHaveValidators [](start = 20, length = 32)

Nice work, James!

One suggestion, you can simply your assembly retrieval by simply finding a type in that project and grabbing the Assembly from it. This will ensure that binary is in memory in all cases, in case of running this test entirely on its own, for example,

typeof(SendGridKeyValidator).Assembly

Reply:
Thanks Michael!

---
In reply to: [586019894](https://github.com/microsoft/sarif-pattern-matcher/pull/250#discussion_r586019894) [](ancestors = 586019894)